src/app/book/book.js

```diff
@@ -1,14 +1,48 @@
-(function () {
+(function() {
   'use strict';
 
-  function BookCtrl ($location) {
+  function BookCtrl() {
+    var date       = $('#date');
+    var startTime  = $('#time-start');
```

Comment:
Fair enough. Didn't know that angular had some jQuery. jQuery was already installed so I figured I'd use that.